### ISSUE TITLE
option in BE for lazy loading images in the gallery content element

### ISSUE THREAD

**Issue by medarob** [benjaminkott/bootstrap_package#839]
# Feature Request
An option for lazy loading images in the new gallery content element.
Right now you can only add a pagination if the number of images exceeds the total number of images per page which can be set in the backend.

## Describe the solution you'd like

A solution would be the new "lazyload"-option which you can be set via an attribute: Currently only supported in Chrome based browsers: https://caniuse.com/#search=lazy

A fall back could be a javascript solution or show a info message which browsers are support. If there is no native browsersupport, fallback to "show all images at once". (If a javascript library as a fallback is used, the library should only be included on the page with the lazyload images.)

**Comment by benjaminkott** [comment]
@medarob all images are already set to loading lazy by default, you mean instead of pagination?

**Comment by medarob** [comment]
Yes, instead of the pagination option or as an additional option.
